Src/WitsmlExplorer.Api/Services/RiskService.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Witsml.Query;
```

Comment:
Same as noted under `CreateRiskWorker.cs`. If possible,  we would like to use `WitsmlExplorer.Api.Query` to be able to build further on the refactoring done after issue #350